quartz_solar_forecast/datapipes/pv_config.yaml

```diff
@@ -0,0 +1,23 @@
+input_data:
```

Comment:
should the name be `pv_and_nwp_config.yaml`?

Reply:
yes, a good suggestion. @peterdudfield, I don't have to upload the data files, right? I added them in `.gitignore`. I can add the links of those files in the comments in the `.yaml` file if anyone wants to try them locally.

Reply:
Yea I think that would be good. Then's it easy for other people to build on it